src/main/resources/bfa-config.properties

```diff
@@ -0,0 +1,2 @@
+webdriver.gecko.driver=/Users/ninhdoan/Desktop/selenium_drivers/firefox/geckodriver
+webdriver.chrome.driver=/Users/ninhdoan/Desktop/selenium_drivers/chrome/chromedriver
```

Comment:
This path has been hard-coding by your local path. Please consider to change for running in other environment.